examples/alloydb-vectorstore-example/alloydb_vectorstore_example.go

```diff
@@ -0,0 +1,200 @@
+	username := os.Getenv("ALLOYDB_USERNAME")
+	if username == "" {
+		log.Fatal("env variable ALLOYDB_USERNAME is empty")
+	}
+	// Requires environment variable ALLOYDB_PASSWORD to be set.
+	password := os.Getenv("ALLOYDB_PASSWORD")
+	if password == "" {
+		log.Fatal("env variable ALLOYDB_PASSWORD is empty")
+	}
+	// Requires environment variable ALLOYDB_DATABASE to be set.
+	database := os.Getenv("ALLOYDB_DATABASE")
+	if database == "" {
+		log.Fatal("env variable ALLOYDB_DATABASE is empty")
+	}
+	// Requires environment variable PROJECT_ID to be set.
+	projectID := os.Getenv("PROJECT_ID")
+	if projectID == "" {
+		log.Fatal("env variable PROJECT_ID is empty")
+	}
+	// Requires environment variable ALLOYDB_REGION to be set.
+	region := os.Getenv("ALLOYDB_REGION")
+	if region == "" {
+		log.Fatal("env variable ALLOYDB_REGION is empty")
+	}
+	// Requires environment variable ALLOYDB_INSTANCE to be set.
+	instance := os.Getenv("ALLOYDB_INSTANCE")
+	if instance == "" {
+		log.Fatal("env variable ALLOYDB_INSTANCE is empty")
+	}
+	// Requires environment variable ALLOYDB_CLUSTER to be set.
+	cluster := os.Getenv("ALLOYDB_CLUSTER")
+	if cluster == "" {
+		log.Fatal("env variable ALLOYDB_CLUSTER is empty")
+	}
+	// Requires environment variable ALLOYDB_TABLE to be set.
+	table := os.Getenv("ALLOYDB_TABLE")
+	if table == "" {
+		log.Fatal("env variable ALLOYDB_TABLE is empty")
+	}
+
+	// Requires environment variable VERTEX_LOCATION to be set.
+	location := os.Getenv("VERTEX_LOCATION")
+	if location == "" {
+		log.Fatal("env variable VERTEX_LOCATION is empty")
+	}
+
+	return username, password, database, projectID, region, instance, cluster, table, location
+}
+
+func main() {
+	// Requires the Environment variables to be set as indicated in the getEnvVariables function.
+	username, password, database, projectID, region, instance, cluster, table, vertexLocation := getEnvVariables()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	pgEngine, err := alloydbutil.NewPostgresEngine(ctx,
+		alloydbutil.WithUser(username),
+		alloydbutil.WithPassword(password),
+		alloydbutil.WithDatabase(database),
+		alloydbutil.WithAlloyDBInstance(projectID, region, cluster, instance),
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Initialize Vectorstore table using InitVectorstoreTable method
+	vectorstoreTableoptions, err := alloydbutil.NewVectorstoreTableOptions(&alloydbutil.VectorstoreTableOptions{
+		TableName:  "table",
+		VectorSize: 768,
+	})
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = pgEngine.InitVectorstoreTable(ctx, *vectorstoreTableoptions,
+		[]alloydbutil.Column{
+			alloydbutil.Column{
+				Name:     "area",
+				DataType: "int",
+				Nullable: false,
+			},
+			alloydbutil.Column{
+				Name:     "population",
+				DataType: "int",
+				Nullable: false,
+			},
+		},
+		"langchain_metadata",
+		alloydbutil.Column{Name: "langchain_id", DataType: "UUID", Nullable: false},
```

Comment:
We don't need to set the default values. please remove

Reply:
on it! will remove them!